articles/application-gateway/application-gateway-faq.md

```diff
@@ -123,7 +123,7 @@ No. Instances are distributed across upgrade domains and fault domains.
 
 ### Does Application Gateway support connection draining?
 
-Yes. You can set up connection draining to change members within a backend pool without disruption. This setup allows you to continue to send existing connections to their previous destination until either that connection closes or a configurable timeout expires. Connection draining waits for only current in-flight connections to finish. Application Gateway isn't aware of the application session state.
+Yes. You can set up connection draining to change members within a backend pool without disruption. For specifics visit [connection draining section of Application Gateway](https://docs.microsoft.com/azure/application-gateway/overview#connection-draining).
```

Comment:
```suggestion
Yes. You can set up connection draining to change members within a backend pool without disruption. For more information, see [What is Azure Application Gateway?](overview.md#connection-draining).
```

Reply:
I believe the link title should indicate the section it is targeting. Or users might not click it thinking that it would just take them to Azure Application Gateway's introduction and not the feature's specification.

Reply:
Taking other changes though.